### DISCUSSION TITLE
G-Helper made my ROG zephyrus run super hot and used tons of battery

### DISCUSSION THREAD

**Discussion by lolmanlol123** [seerge/g-helper#3332] [General]
Idk if this is just me or but..
![image](https://github.com/user-attachments/assets/16d4c01a-7cc6-4a41-957a-fcc7678f0cfb)

**Comment by lolmanlol123** [comment]
^ left my pc out for like 30m with nothing at all open and when i came back it was at 9 percent

**Comment by seerge** [comment]
Dear @lolmanlol123,

Your device can get hot when it runs some tasks, and therefore consumes power, including system own (like windows updates / antivirus scanes / etc) that will not appear under "Battery usage per app" and not specifically because of G-Helper. You can check processes running in windows by opening Task Manager and sorting them by CPU usage.

If you have specific problem with G-Helper you need [to post and issue](https://github.com/seerge/g-helper/issues) and explain it in high details, including app logs and other requested information. Without that it's impossible to assist you further.

Thanks.
